js/livecoin.js

```diff
@@ -356,10 +356,12 @@ module.exports = class livecoin extends Exchange {
         };
     }
 
-    async cancelOrder (id, symbol = undefined, params = {}) {
+    async cancelOrder (id, symbol, params = {}) {
```

Comment:
Hi! First of, thanks for your contribution!

Unfortunately, we can't merge that definition... Despite it does work in JS, it will break PHP and Python inheritance/override chain with a warning...

We need to keep that signature as it was: `async cancelOrder (id, symbol = undefined, params = {}) {` and to check if that symbol is defined in the method's body, like so:

```
async cancelOrder (id, symbol = undefined, params = {}) {
    if (!symbol)
        throw new ExchangeError (this.id + ' cancelOrder requires a symbol argument');
    await loadMarkets ();
    let market = this.market (symbol);
    ...
}
```

↑ Can you do that? Thx!